crates/sui-indexer-alt/migrations/2024-10-28-144002_sum_coin_balances/up.sql

```diff
@@ -0,0 +1,20 @@
+-- A summary table for coins owned by addresses
+--
+-- This can be used to paginate the coin balances of a given address at an
+-- instant in time, returning coins in descending balance order.
+CREATE TABLE IF NOT EXISTS sum_coin_balances
```

Comment:
naive q, why `sum_` instead of just `coin_balances`?

Reply:
`sum` stands for "summary", it's the equivalent of the "_snapshot" from "objects_snapshot". There will be an additional "wal_coin_balances" that is like the old "objects_history".